api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -62,13 +62,10 @@ public void validate(Object object, Errors errors) {
 		if (log.isDebugEnabled()) {
 			log.debug(this.getClass().getName() + ".validate...");
 		}
-		
-		if (object == null) {
-			throw new IllegalArgumentException("The personAddress object should not be null");
-		}
-		
 		PersonAddress personAddress = (PersonAddress) object;
-		
+		if (personAddress == null) {
+           errors.rejectValue("value", "error.null");
+		}
 		//resolve a shorter name to display along with the error message
 		String addressString;
 		if (StringUtils.isNotBlank(personAddress.getAddress1())) {
```

Comment:
Rename function name getAddress1 to more self-explanatory name

Reply:
@saadkhaleeq610 Did you see the PR title? Renaming functions is not part of it.